README.md

```diff
@@ -15,71 +15,75 @@ Here lies our Discord emojis!
+| ![](emoji/lookup.png)               | :lookup:                | Everfree Northwest Art ([JaDeDJynX](https://www.deviantart.com/jadedjynx)) |
+| ![](emoji/marina.png)               | :marina:                | Everfree Northwest Art ([JaDeDJynX](https://www.deviantart.com/jadedjynx)) |
+| ![](emoji/marinaAngry.png)          | :marinaangry:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaEep.png)            | :marinasilly:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaEugh.png)           | :marinaeugh:            | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaRage.png)           | :marinarage:            | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaSad.png)            | :marinasad:             | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaShock.png)          | :marinashock:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaSilly.png)          | :marinasilly:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaSmile.png)          | :marinasmile:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marina_high5_l.png)       | :marina_high5_l:        | FearDaKez |
+| ![](emoji/marina_high5_r.png)       | :marina_high5_r:        | FearDaKez |
+| ![](emoji/marinafacehoof.png)       | :marinafacehoof:        | [imdrunkonTea](https://www.deviantart.com/imdrunkontea) |
+| ![](emoji/marinashroom.png)         | :marinashroom:          | [imdrunkonTea](https://www.deviantart.com/imdrunkontea) |
+| ![](emoji/marinasparkle.png)        | :marinasparkle:         | FearDaKez |
+| ![](emoji/marinawink.png)           | :marinawink:            | Everfree Northwest Art ([dawnfire](https://www.deviantart.com/dawnf1re)) |
+| ![](emoji/mocha.png)                | :mocha:                 | Everfree Northwest Art |
+| ![](emoji/news.png)                 | :news:                  | http://fav.me/d4unq6t |
+| ![](emoji/nope.png)                 | :nope:                  | |
+| ![](emoji/novella.png)              | :novella:               | Everfree Northwest Art |
+| ![](emoji/party.png)                | :party:                 | https://derpibooru.org/images/85252 cropped |
+| ![](emoji/rekt.png)                 | :rekt:                  | |
+| ![](emoji/report.png)               | :report:                | Everfree Northwest Art ([Dana Simpson](https://danasimpson.com/))|
+| ![](emoji/sad.png)                  | :sad:                   | |
+| ![](emoji/sharpie.png)              | :sharpie:               | Everfree Northwest Art ([JaDeDJynX](https://www.deviantart.com/jadedjynx)) |
+| ![](emoji/shrug-1.png)              | :shrug-1:               | |
+| ![](emoji/sick.png)                 | :sick:                  | |
+| ![](emoji/silverscroll_high5_l.png) | :silverscroll_high5_l:  | FearDaKez |
+| ![](emoji/silverscroll_high5_r.png) | :silver scroll_high5_r: | FearDaKez |
```

Comment:
slight preference to rename to `silver_high5_l/r` or `silverscroll_h5l/r` to not extend table / have long emoji names.